evalml/model_understanding/prediction_explanations/explainers.py

```diff
@@ -195,3 +196,52 @@ def explain_predictions_best_worst(pipeline, input_features, y_true, num_to_expl
                                              output_format=output_format, top_k_features=top_k_features,
                                              include_shap_values=include_shap_values, num_to_explain=num_to_explain)
     return report_creator(data)
+
+
+def clean_format_tree(clf):
```

Comment:
@freddyaboulton Even though it isn't exactly an intermediate function to `visualize_decision_tree`, I agree that it would make more sense in `model_understanding/graphs.py`! And I agree with `get_tree_data` since it's definitely more in line with convention. Thanks!